doc/source/whatsnew/v0.22.0.txt

```diff
@@ -23,7 +23,7 @@ Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
 - Better support for ``Dataframe.style.to_excel()`` output with the ``xlsxwriter`` engine. (:issue:`16149`)
--
+- :func:`pd.tseries.frequencies.to_offset()` now accepts leading '+' signs e.g. '+1h'. (:issue:`18171`)
```

Comment:
I think this needs to be pandas (and not pd)

Reply:
The other entries in the file all say 'pd' and not 'pandas'. In previous whatsnew.txt files there seems to be a mixture but still a preponderance of 'pd'

Reply:
hmm, then maybe this works. ok.